<!--xml-->
<launch>
    <node name="joy_node" pkg="joy" type="joy_node"  />
   <node name="usercontrol_node" pkg="icarus_rover_v2" type="usercontrol_node"  output="screen">
        <param name="loop_rate"       value="100"/>
	    <param name="verbosity_level" value="DEBUG"/>
        <param name="require_pps_to_start" value="false" />
        <param name="Operation_Mode"    value="Arm" />
  </node>
</launch>
